src/overlays/actors/ovl_Obj_Grass/z_obj_grass.c

```diff
@@ -56,35 +58,35 @@ static ColliderCylinderInit sCylinderInit = {
     { 6, 44, 0, { 0, 0, 0 } },
 };
 
-Vec3f D_809AAB4C[] = {
-    { 0.0f, 0.707099974155f, 0.707099974155f },
-    { 0.707099974155f, 0.707099974155f, 0.0f },
-    { 0.0f, 0.707099974155f, -0.707099974155f },
-    { -0.707099974155f, 0.707099974155f, 0.0f },
+static Vec3f sUnitDirections[] = {
+    { 0.0f, 0.7071f, 0.7071f },
+    { 0.7071f, 0.7071f, 0.0f },
+    { 0.0f, 0.7071f, -0.7071f },
+    { -0.7071f, 0.7071f, 0.0f },
```

Comment:
Out of curiosity does `M_SQRT1_2` work for the `0.7071`s?

Reply:
nope, you need way too many digits of precision I'd assume